Earthfile

```diff
@@ -366,11 +367,20 @@ earthly-integration-test-base:
         END
     ELSE
     # Use a mirror, supports mirroring Docker Hub only.
+        IF [ "$DOCKERHUB_MIRROR_INSECURE" = "true" ]
+            ARG _MIRROR_CONFIG="[registry.\"$DOCKERHUB_MIRROR\"]
+                                http = true
+                                insecure = true"
+        ELSE
+            ARG _MIRROR_CONFIG=""
+        END
         ENV GLOBAL_CONFIG="{disable_analytics: true, local_registry_host: 'tcp://127.0.0.1:8371', conversion_parallelism: 5, buildkit_additional_config: '[registry.\"docker.io\"]
 
-                           mirrors = [\"$DOCKERHUB_MIRROR\"]'}"
+                           mirrors = [\"$DOCKERHUB_MIRROR\"]
+                           $_MIRROR_CONFIG'}"
         ENV EARTHLY_ADDITIONAL_BUILDKIT_CONFIG="[registry.\"docker.io\"]
-                    mirrors = [\"registry-1.docker.io.mirror.corp.earthly.dev\"]"
```

Comment:
I was hesitant to change this, given that `DOCKERHUB_MIRROR` was introduced just a few lines up (suggestion this was left untouched on purpose).  But I also couldn't see any obvious reason why that specific value was required (unless there's some test, possibly of the release process itself, that requires it).